### PR TITLE
Visual keys

### PR DESCRIPTION
**Pull request by rpmohn**
Hi Marc,

I submitted this through email about a year ago, but I'm trying out the GitHub workflow now. 

This branch is to display command keypresses in the status bar while a multiple keypress command is being entered. Almost all commands in dvtm begin with MOD, so require multiple keypresses. Command keys are displayed as they are pressed, and appear between the layout symbol and the status text. The keys are cleared from the status bar after the final key of a command is pressed, or if the key combination is not bound to a command. For example, if your MOD is set to ^G and you are toggling the view of tag 4, it first displays '^g' after you press MOD, then '^gV' after you press 'V', and then is cleared after you complete the command by pressing '4'.

Thanks -Ross

### CODE REVIEW

**Comment by josuah** [comment]
I already saw this patch for dvtm on your blog.  I will already adopt this patch, thank you!

**Comment by rpmohn** [comment]
I'd like to make another request that this patch be merged into the master branch. I think this is a good change and that everyone would like to see it. Thank you! -Ross

**Comment by rpmohn** [comment]
Here's an example of what it looks like. On the upper right, you can see that I've pressed '^g' as my MOD key, then 'v', so now I'm ready to select the number of the view I'll switch to. This is very similar to the way in which vim displays key presses as you type them, until you complete a full command. 

screenshot - 09132018 - 06 44 35 pm

File: dvtm.c [hunk 1]
```diff
@@ -345,6 +346,15 @@ drawbar(void) {
 	attrset(TAG_NORMAL);
 	addstr(layout->symbol);
 
+	if(keys) {
```
Comment (KAction) [inline dvtm.c:358]: missing space after if

**Review by KAction — CHANGES_REQUESTED** [review]

File: dvtm.c (hunk 1 above)
Comment (rpmohn) [reply dvtm.c:358]: Sure, and I also changed it to display an uppercase mod key, to match vi, so 
`printw("^%c", 'A' - 1 + keys[keycount++]);`

Comment (KAction) [reply dvtm.c:358]: Oh, and looks like `if (keys)` condition is no-op. Check https://builds.sr.ht/~kaction/job/449596.

Comment (martanne) [reply dvtm.c:358]: This should probably be `if (keys[0])`.

**Comment by martanne** [comment]
Not sure whether I like that there is no separation between the layout symbol and the pending input, but I will apply it with some cosmetic changes.

File: dvtm.c (hunk 1 above)
Comment (KAction) [reply dvtm.c:358]: It looks like it works as-is, so probably this conditional can be skipped.

Comment (KAction) [reply dvtm.c:358]: It is already covered by "while (keycount < MAX_KEYS && keys[keycount])
" loop

**Pull request closed without merging**